proposals/model-management.md

```diff
@@ -0,0 +1,67 @@
+# Model Management
+
+## Problem
+
+This proposal is attempting to solve the following problems:
+
+* Slow model loading on cold-start
+* Storage of LoRA adapters generated from finetuning jobs
+* Dynamic LoRA adapter injection
+
+While working within the following constraints:
+
+* KubeAI should be able to run with zero external dependencies
+* KubeAI should be able to run on a local machine
+* KubeAI should be able to be run on clouds with heterogenous disk features (limited support for ReadOnlyMany)
+
+## Solution
+
+KubeAI should support configurable storage for both models and adapters. These settings should be configurable at the `kind: Model` level as some variants (like GCP Hyperdisk ML) are expensive and generally only recommended for larger models.
+
+```yaml
+kind: Model
+spec:
+  storageProfile:
```

Comment:
is StorageProfile optional? Can we just leave it out and let it pull from HF?

Reply:
For pulling models, yes, thinking we have a storage profile option of `None`. I added a block for default storage profiles when none are specified on the Model.